docs/src/pages/demos/steppers/steppers.md

```diff
@@ -68,6 +68,10 @@ This component uses a customized `StepConnector` element that changes border col
 
 {{"demo": "pages/demos/steppers/CustomizedStepper.js"}}
 
+## Breadcrumbs
+
```

Comment:
I'd add a description here along the lines of: "The stepper components can be repurposed to create navigation breadcrumbs."

Trivia - breadcrumbs were one of the first web-server plugin components I ever created (back in 1998), but I didn't know they were called that then, so it was called "clickable path".

Reply:
Done :)

I did some digging to try to find the origin and also found another alias: "Cookie crumbs" (https://en.wikipedia.org/wiki/Breadcrumb_(navigation)).

Reply:
I also did some digging, and would you believe, it's still out there!

http://source.riverweb.com/modules/modinfo.html?id=37

This was for Roxen, a web server that was *so* far ahead of its time. You could add plugins ("modules"), and their configuration page would show up in the web server's web GUI. It supported "RXML" which allowed you to embed tags in your html pages, that would then be converted by the respective module to HTML, images, etc. when the page was requested.

Sadly Apache won out, at the time, and Roxen went on to become a niche CMS.